vault/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,48 @@
+name: Vault
+files:
+- name: vault.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/http
+  - template: instances
+    options:
+    - name: api_url
+      required: true
+      description: URL of the Vault to query.
+      value:
+        example: http://localhost:8200/v1
+        type: string
+    - name: detect_leader
+      required: false
+      description: Whether or not this instance should report cluster leader change events.
+      value:
+        example: false
+        type: boolean
+    - name: client_token
+      required: false
+      description: Client token necessary to collect metrics.
+      value:
+        example: <CLIENT_TOKEN>
+        type: string
+    - name: client_token_path
+      required: false
+      description: |
+         Path to a file containing the client token. Overrides `client_token`.
+         The token will be re-read after every authorization error.
+      value:
+        example: <CLIENT_TOKEN_PATH>
+        type: string
+    - template: instances/http
+    - template: instances/tags
+    - template: instances/global
+  - template: logs
+    example:
+    - type: file
+      path: /vault/vault-audit.log
+      source: vault
+      service: vault
```

Comment:
```suggestion
      service: <SERVICE_NAME>
```

Reply:
@FlorianVeaux I noticed we've been switching to this style in several other integration docs (and we use it throughout the example config here), but do we want to keep going in that direction? When enabling logs locally we'll be sending `service:<SERVICE_NAME>` tags to the logs backend, and I'm not sure this is valid…

Reply:
Yes to remove friction in setting up logs I'd prefer to default to the correct/expected name, in this case `vault`

Reply:
@ofek @florimondmanca 

We expect users to configure this field, keeping `service: <SERVICE_NAME>` reduces friction for users IMO as they won't forget to configure this field properly. 

I don't think our testing process should be of any importance here. If we're unhappy with how enabling logs locally for testing purposes is sending a useless `service` name, we can always add the necessary overlay to make it configurable. In any case I think this should be considered minor compared to user experience.